### ISSUE TITLE
Improve the country coverage chart

### ISSUE THREAD

**Issue by hellais** [ooni/explorer#687]

**Comment by d80ep08th** [comment]
I am interested. Can you please tell me wjat the path to that page is ?

**Comment by girishsontakke** [comment]
@hellais is there any country for which sufficient OONI Probe measurement coverage is available?

**Comment by hellais** [comment]
Yes the path to the country page is the following: https://explorer.ooni.org/country/IT.

It's probably helpful if I share a bit more insight into what I was thinking of in terms of improvements to this view.

At a high level, what we are trying to convey through this chart is "how well do we cover with OONI measurements this particular country".

In terms of coverage for a particular country, I think the main ways in which we have been slicing it is:
- What is the absolute count of measurements
- How many networks do we have insight into

In the current chart we are basically collapsing both of these into one, but I feel like the chart doesn't do a very good job at communicating the interpretation of this data. If you just look at the absolute values, it's unclear if the numbers you are seeing are good or bad, because there is no point of reference.

Moreover the chart is able to display information from the last month, so it's not providing longitudinal information on the coverage over time.

What I was thinking of is that we could simplify this a bit and use a chart like this one: https://nivo.rocks/calendar/, where we define for the absolute count of measurements 4 ranges, that have an associated color:
- [0 - k1] -> data absent/very poor coverage
- [k1 - k2] -> poor coverage
- [k2 - k3] -> good coverage
- [k3 - k4] -> great coverage

The tricky part is coming up with reasonable values for k1,k2,k3,k4 and evaluating if these should change over time or we should just hardcode them in the client.
The other question is, should these values change depending on the country in question? For example it might be the case that some number of measurements is enough to have a good coverage in a small country with few networks, but that value may be insufficient for countries with more networks.
A good starting point could be looking at the .25, .50, .75, .90 quantiles of measurement counts by country and seeing if using those as ranges leads to something useful.
This can be computed quite easily using the data made available by the aggregation API: https://api.ooni.io/api/v1/aggregation?axis_y=probe_cc&axis_x=measurement_start_day&since=2022-01-03&until=2022-01-05

For the ASN coverage we would do something similar, yet in this case I think the values must somehow take into account the number of networks that exist in a particular country. For example we can use the number of ASNs that are announcing prefixes (from something like RIPE: https://stat.ripe.net/ui2013/IT#tabId=database) and use some factor of the ones we see measurements for vs the total number (it's important to keep in mind that many of these ASs might not be residential ones, so we aren't actually aiming for ever reaching 100%).

**Comment by Mohamed-Ayaan358** [comment]
@hellais, just to confirm 
![image](https://user-images.githubusercontent.com/72858215/161679737-42a03450-9cf8-4546-af6e-9cdc81cc3725.png)
This is the chart that we are discussing? And if so may I take a shot at solving this issue?

**Comment by hellais** [comment]
Hi @Mohamed-Ayaan358 , the chart I'm talking about here is actually the one at the beginning of the country page:
![Screenshot 2022-04-05 at 11 36 51](https://user-images.githubusercontent.com/424620/161725383-4f683447-edb5-4247-8a3d-55d29175928a.png)

**Comment by agrabeli** [comment]
This is also related to: https://github.com/ooni/explorer/issues/266#issuecomment-1102860915 (displaying monthly and yearly coverage per country)
